### ISSUE TITLE
Metadata of cells3d example dataset appears incorrect when opening it with Fiji.

### ISSUE THREAD

**Issue by haesleinhuepf** [scikit-image/scikit-image#6178]
## Description

Hi all,

I just noticed that the meta-data for the "cells3d" example appears not properly configured if I open the dataset in ImageJ/Fiji. It looks like this:

![image](https://user-images.githubusercontent.com/12660498/148938799-ad9f7ce4-172a-4762-8b01-f40337ae9701.png)

I could resave the dataset using Fiji so that it contains the right meta data. The pixel size is `(0.29 0.26 0.26)` microns in Z-Y-X according to [this description](https://scikit-image.org/docs/dev/api/skimage.data.html#skimage.data.cells3d). Just let me know if it makes sense and I'm happy to send a PR to the gitlab server.

Thanks!
Robert

## Way to reproduce

Download https://gitlab.com/scikit-image/data/-/blob/master/cells3d.tif open it in [Fiji](https://fiji.sc/Downloads). CLick on the menu `Image > Properties...` to open the dialog shown above.

## Version information

![image](https://user-images.githubusercontent.com/12660498/148939016-38560c0e-1a3e-4153-82b1-d1861ffa0f0b.png)

**Comment by mkcor** [comment]
Apparently the license for this data file is [unknown](https://gitlab.com/scikit-image/data), so let's hope the [Allen Institute for Cell Science](https://alleninstitute.org/what-we-do/cell-science/about/) don't mind us reusing and resaving the file with corrected metadata. I assume they won't.

Thank you, @haesleinhuepf! Waiting for your PR on GitLab :heart:

**Comment by alexdesiqueira** [comment]
Hey @mkcor,
> Apparently the license for this data file is unknown

that's weird, thanks for pointing that out. I'll open an issue to investigate it.

**Comment by haesleinhuepf** [comment]
Alright, thanks for the feedback @alexdesiqueira  and @mkcor. Here you go:
https://gitlab.com/scikit-image/data/-/merge_requests/17

**Comment by grlee77** [comment]
Specifically, we have the SHA256 hash stored here: https://github.com/scikit-image/scikit-image/blob/ba987180c1c6145896c7f10b423199dc34888039/skimage/data/_registry.py#L156
We could easily update that in the development branch to match the file with the fixed metadata, but that doesn't help all of the existing scikit-image installations which till use the old hash value and would fail on download with
```
ValueError: SHA256 hash of downloaded file (cells3d.tif) does not match the known hash: expected afc7c7d80d38bfde09788b4064ac1e64ec14e88454ab785ebdc8dbba5ca3b222 but got bd35a300fbe4c90da5707eaa8b08f8893cb00787d81371c339057e068589691c. Deleted download for safety. The downloaded file may have been corrupted or the known hash may be outdated.
```

One workaround would be to save the version with fixed metadata under a new name (`cells3d_fixed.tiff`) and then update the `skimage.data.cells3d` function here to use that filename instead for future releases. Existing installations would continue to use the original file, but at least new versions would have the fixed metadata. Not sure if there are other reasonable workarounds?

**Comment by mkcor** [comment]
Right, we need a different filename since the file is different: I know it's verbose, but I would suggest `cells3d_correct_metadata.tif` (so it's explicit; and end users don't see this name anyway).

Yes, we need both files, old and new, in the data registry. Would it make sense to 'deprecate' `cells3d.tif` ultimately? The warning message should then be backported to v0.19, v0.18, ...

**Comment by jni** [comment]
Let's just leave the old file and versions around, and focus on the future. 😊
